src/App.h

```diff
@@ -153,6 +153,13 @@ struct TemplatedApp {
         httpContext->filter(std::move(filterHandler));
     }
 
+    /* Publishes a message to all websocket contexts - conceptually as if publishing to the one single
+     * TopicTree of this app (technically there are many TopicTrees, however the concept is that one
+     * app has one conceptual Topic tree) */
+    bool publish(std::string_view topic, std::string_view message, unsigned char opCode, bool compress = false) {
```

Comment:
this is dumb and as discussed shouldn't include it

Reply:
right :)